### PR TITLE
GSoC 2022 blog post

### PR DESCRIPTION
**Pull request by IAlibay**
This will be the 2022 GSoC projects announcement posts.

We will be asking @aya9aladdin, and @BFedder to fill in the details of the posts (please have a look at this file for inspiration on how to complete the details: https://github.com/MDAnalysis/MDAnalysis.github.io/blob/master/_posts/2021-05-19-gsoc-students.md

Note - adding a picture of yourself is not mandatory, if you feel uncomfortable with this we can either replace it with an avatar picture or just not have one at all.

### CODE REVIEW

File: _posts/2022-05-27-gsoc-students.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Write something about your project and yourself here (don't forget to link to the project page on the GSoC website)!
+
+Add where we can find you here! (github, twitter, others)
+
+Add a link to your blog here!
+
+
+— @jbarnoud @hmacdope @ojeda-e @IAlibay @fiona-naughton @orbeckst @lilyminium (mentors)
```
Comment (IAlibay) [inline _posts/2022-05-27-gsoc-students.md:39]: I hope I haven't forgotten someone we have so many mentors 🙈 

@richardjgowers and @PicoCentauri you weren't on the mentor list for either students, would you still like to be involved as a mentor? (we can still add you as an extra mentor on either projects or you can be "unofficial" mentors if you'd like).

File: _posts/2022-05-27-gsoc-students.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+layout: post
+title: Google Summer of Code Students 2022
+---
+
+We are happy to announce that MDAnalysis is hosting two [GSoC][gsoc]
+students this year -- @aya9aladdin, and @BFedder. MDAnalysis has been accepted as its own
+[organization with GSoC][mda-gsoc] for the third year running and we are grateful to Google for granting us
+the opportunity to undertake _two very exciting GSoC projects_!
+
+## Aya Mohamed Alaa: Context-aware guessers
```
Comment (IAlibay) [inline _posts/2022-05-27-gsoc-students.md:11]: I haven't put down the lengths of each project - 12 weeks of Aya's and 6 weeks for Bjarnes I believe?

Given that we still have yet to discuss if the projects will be taken on a part-time basis, I didn't think it was appropriate to write it down here.

**Comment by aya9aladdin** [comment]
Great! I'm working on it

**Comment by orbeckst** [comment]
Skeleton looks good to me—good to have the students already link to their blogs so that they set them up already. 

When do we launch our blog post, ie what’s the deadline for @aya9aladdin and @bfedder to add their content?

**Comment by IAlibay** [comment]
So tentatively next a Friday (hence the date on the posts), that being said I'll be emailing them both tomorrow with a formal hello + Todo list, it may be that we want to wait until after we have a first chat.

**Comment by orbeckst** [comment]
@BFedder 's contribution is in PR #229 and can be merged into this one.

@aya9aladdin please get your update done soon. If you have any questions, please ask.

**Comment by aya9aladdin** [comment]
Sorry for the late reply! I had some issues with my laptop in the last week and was migrating to my new laptop that's why I wasn't available. I have just committed my introductory section of the post, hope I did it the right way

**Comment by orbeckst** [comment]
@aya9aladdin I reviewed your PR #230 ; once you've addressed the comments please say something so that either I or @IAlibay can merge it. Thanks!

**Review by IAlibay — COMMENTED** [review]
I can't self-approve, but this lgtm.

@orbeckst are we good to merge?

**Review by orbeckst — APPROVED** [review]
Lgtm

**Comment by orbeckst** [comment]
Post is up https://www.mdanalysis.org/2022/05/31/gsoc-students/

**Comment by IAlibay** [comment]
We'll need to tweet this out if anyone has the time to do so.

**Comment by orbeckst** [comment]
🐦 ed https://twitter.com/mdanalysis/status/1531703351426985984

**Pull request merged**